### PR TITLE
Added a Chat Abbrevation cheatsheet

### PR DESCRIPTION
**Pull request by Midhun-Jo-Antony**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New {chat abbreviation} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

reference to chat abbrevations

---
https://duck.co/ia/view/chat_abbrevations_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@Midhun-Jo-Antony Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by gaulrobe** [comment]
@Midhun-Jo-Antony not sure why you closed #3222, but be sure to review my comments there 👍

**Comment by Midhun-Jo-Antony** [comment]
@gaulrobe Sure i will do that

File: share/goodie/cheat_sheets/json/chat-abbrevations.json [hunk 1]
```diff
@@ -0,0 +1,172 @@
+{
+	"id": "chat_abbrevations_cheat_sheet",
+
+
+	"name": "chat Abbreviation",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/chat-abbrevations.json]: Chat Abbreviation would be good

**Comment by zekiel** [comment]
@Midhun-Jo-Antony thanks for keeping at this. We'll get this up on beta soon!

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/chat_abbrevations_cheat_sheet_mobile.png?E0WpXXKWiK)

**Comment by tagawa** [comment]
Thanks for waiting @Midhun-Jo-Antony - this is now on the beta server for testing: https://beta.duckduckgo.com/?q=chat+cheat+sheet&ia=cheatsheet&iax=1

**Comment by GuiltyDolphin** [comment]
@Midhun-Jo-Antony Don't we already have a cheat sheet for this? https://duck.co/ia/view/sms_cheat_sheet - perhaps you'd like to extend the triggers and improve that IA?

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=chat cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/chat_abbrevations_cheat_sheet_mobile.png?pD7wuUc13F)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=chat cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/chat_abbrevations_cheat_sheet_mobile.png?Ayjn8WtQeQ)

**Comment by zekiel** [comment]
@Midhun-Jo-Antony  since this already exists at https://duck.co/ia/view/sms_cheat_sheet, do you want to submit a PR against that instant answer? That is, if there's anything not included there already.

**Comment by Midhun-Jo-Antony** [comment]
hai, #3223 is not there in the duckduckgo as a cheat sheet. Please review this and make some help for completing this

**Comment by zekiel** [comment]
@Midhun-Jo-Antony Sorry for the confusion. There is already an [SMS chat Cheat Sheet](https://duckduckgo.com/?q=sms+cheat+sheet&ia=cheatsheet&iax=1) so you could just update that Instant Answer with whatever's missing. 

Does that makes sense? Otherwise, we can definitely merge this -- but there will be 2 IAs that do pretty much the same thing.

**Comment by GuiltyDolphin** [comment]
@zekiel We should probably refrain from adding cheat sheets that do the same thing as existing cheat sheets - otherwise things could get a bit out of hand.

@Midhun-Jo-Antony There's a couple of things to do in order to implement your additions on top of the existing cheat sheet:

1. Add aliases e.g, `chat abbreviations` (the name of this cheat sheet)
2. Add any fields from this cheat sheet *that aren't already in the SMS cheat sheet*

@gmahesh23 Got any input on this?

**Comment by GuiltyDolphin** [comment]
@Midhun-Jo-Antony Closing this in favor of the existing Instant Answer.

**Pull request closed without merging**